src/infra/main/template.yaml

```diff
@@ -1082,3 +1284,8 @@ Outputs:
   SqsQueueUrl:
```

Comment:
Could we rename the to TxMAEgressSQSQueueUrl

Reply:
Done - I checked alarm and core template and that value is not imported there, so no further changes as far as I could see. But please let me know if you that is imported somewhere and we need to update